charts/kong/README.md

```diff
@@ -234,6 +234,7 @@ Kong can be configured via two methods:
 | plugins                            | Install custom plugins into Kong via ConfigMaps or Secrets                            | `{}`                |
 | env                                | Additional [Kong configurations](https://getkong.org/docs/latest/configuration/)      |                     |
 | runMigrations                      | Run Kong migrations job                                                               | `true`              |
+| migrationAnnotations               | Annotations for migration jobs                                                        | `{"sidecar.istio.io/inject": "false", "kuma.io/sidecar-injection": "disabled"}` |
```

Comment:
Question:

Did you consider doing instead?
```yaml
migration:
  annotations: {}
```

We already have `runMigrations` as another top-level field.

Reply:
If the migration configuration gets more complex I think it'd make sense to do so. My rationale was that it'd be weird to have
```
runMigrations: true
migrations:
  annotations: {}
```
with a split between some config under `migrations` and some under the root. We could move both under, but it'd be a breaking change or require that we support both `runMigrations` or `migrations.enabled`.

IMO two settings is small enough that I don't really mind both being at the root. Do you think we should restructure or now, or fine to let it wait until we have more in the future?

One other thing to consider is that the plans for split deployments touch this also, e.g. incorporating those we'd maybe have:
```
migrations:
  init: true
  pre-upgrade: true
  post-upgrade: true
  annotations: {}
```
That's more breaking, but it's needed anyway. Should we try for that, add a deprecation warning for runMigrations, and release both as part of 1.5?

Reply:
This sounds good to me. Let's do this.